libs/contact/testing/src/drivers/testing/testing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { DaffContactDriver } from '@daffodil/contact';
+import { DaffTestingContactService } from './contact.service';
+
+@NgModule()
+export class DaffContactTestingDriverModule {
+  static forRoot(): ModuleWithProviders {
+    return {
+      ngModule: DaffContactTestingDriverModule,
+      providers: [
+        {
+          provide: DaffContactDriver,
+          useExisting: DaffTestingContactService
+        }
+      ]
+    };
+  }
+}
```

Comment:
I think you're going to want one of these for the inmemory driver too

Reply:
Good call, addressed in cd4422f5ca68cd864ae0fb22902b7a88e7ef84cc